clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -498,16 +499,22 @@ private UpdateMetadataRequest createUpdateMetadataRequest(int version, String ra
         partitionStates.put(new TopicPartition("topic20", 1),
                 new PartitionState(1, 0, 1, new ArrayList<>(isr), 2, new HashSet<>(replicas)));
 
-        Map<SecurityProtocol, UpdateMetadataRequest.EndPoint> endPoints1 = new HashMap<>();
-        endPoints1.put(SecurityProtocol.PLAINTEXT, new UpdateMetadataRequest.EndPoint("host1", 1223));
+        SecurityProtocol plaintext = SecurityProtocol.PLAINTEXT;
```

Comment:
I know. :) I haven't pushed the test changes as those are still in progress (as I said in a PR comment that probably got lost in the Jenkins bot messages).